src/main/webapp/item-page.html

```diff
@@ -0,0 +1,36 @@
+<!--
+Copyright 2019 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Title ...</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="/css/user-page.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBQKQXGIYnT1dc3c09KXic9SgBteMtb7No"></script> 
+    <script src="/js/Item_page_loader.js"></script>
```

Comment:
Why is Item capitalized here?

Reply:
somehow js let me get away with this.. fixed now